Update / create keyboard support topics (#2002)
DevExpress/devextreme-documentation: concepts/05 UI Components/Lookup/30 Keyboard Support.md

        <td>Tab / Shift + Tab</td>
        <td>If the drop-down menu is opened, moves focus to the list / search field.</td>
    </tr>
    <tr>
        <td>Page Up / Page Down</td>

>         <td>Moves focus to the first/last item of the list's page.</td>

Reviewer: ```suggestion
        <td>Moves focus to the first/last menu item on the page.</td>
```

    </tr>
    <tr>
        <td>Home / End</td>

>         <td>Moves focus to the list's first/last item.</td>

Reviewer: ```suggestion
        <td>Moves focus to the first/last menu item.</td>
```
